src/main/java/com/microsoft/azure/relay/RelayConnectionStringBuilder.java

```diff
@@ -144,7 +144,7 @@ public String toString() {
 
 		if (this.operationTimeout != RelayConstants.DEFAULT_OPERATION_TIMEOUT) {
 			connectionStringBuilder.append(OPERATION_TIMEOUT_CONFIG_NAME + KEY_VALUE_SEPARATOR
-					+ this.operationTimeout.toString() + KEY_VALUE_PAIR_DELIMITER);
+					+ this.operationTimeout.getSeconds() + KEY_VALUE_PAIR_DELIMITER);
```

Comment:
>getSeconds [](start = 29, length = 10)

Just to be certain, does this return a long/int? Just don't want it to ever have a fractional component.